examples/resources/elasticstack_kibana_maintenance_window/import.sh

```diff
@@ -0,0 +1 @@
+terraform import elasticstack_kibana_maintenance_window.my_maintenance_window <space id>/<maintenance window id>
```

Comment:
This command will not work for the resource as it doesn't implement the import function. 

Terraform import is not supported by default, it has to be implemented in the resource:

https://github.com/elastic/terraform-provider-elasticstack/blob/4e803b236197eb784d94c4af1e464d012f595f25/internal/kibana/data_view/resource.go#L37-L53

Reply:
Fixed in [0d590bc](https://github.com/elastic/terraform-provider-elasticstack/pull/1224/commits/0d590bce4dccc978702a39d0f64527523ac26036).